Jenkinsfile

```diff
@@ -14,29 +14,8 @@ pipeline {
             steps {
                 checkout scm
 
-                sh 'mkdir -p .build'
+                sh './scripts/build.sh'
 
-                sh 'service postgresql start'
-            }
-        }
-
-        stage('Verification') {
-            parallel {
-                stage('Testsuite') {
-                    environment {
-                        STORJ_SIM_POSTGRES = 'postgres://postgres@localhost/teststorj?sslmode=disable'
-                    }
-                    steps {
-                        sh 'psql -U postgres -c \'create database teststorj;\''
-                        sh './scripts/test-java.sh'
```

Comment:
If this is gone then what is running the tests?

Reply:
Github action runs all the tests (including the examples).

Reply:
I don't see the output. Do we need to change some repo setting?